src/cli.ts

```diff
@@ -40,6 +41,12 @@ async function main() {
 				hidden: true,
 				default: process.env["TMP_JYC"] === "true",
 			},
+			custom_decorators: {
+				type: "string",
+				hidden: true,
+				default: process.env["CUSTOM_DECORATORS"] ?? "{}",
+				description: `Example CUSTOM_DECORATORS = '{"@Fields.string":"@KitFields.string#@kitql/remult"}', it will be JSON parsed!`,
```

Comment:
i think it needs more docs
for example explain the syntax
why should the exist